gtsam/geometry/SimpleCamera.h

```diff
@@ -33,7 +34,8 @@ namespace gtsam {
   using PinholeCameraCal3Bundler = gtsam::PinholeCamera<gtsam::Cal3Bundler>;
   using PinholeCameraCal3DS2 = gtsam::PinholeCamera<gtsam::Cal3DS2>;
   using PinholeCameraCal3Unified = gtsam::PinholeCamera<gtsam::Cal3Unified>;
-
+  using PinholeCameraCal3Fisheye = gtsam::PinholeCamera<gtsam::Cal3Fisheye>;
```

Comment:
Hmmm. why do we even have these typedefs and includes here? Seems like it exacerbates header bloat/compile times.

Reply:
My guess is that it is an artifact of before we understood the wrapper's behavior really well. The wrapper has seen some nice improvements in the past year so we may as well kill this, while moving the typedef for `Cal3_S2` inside the deprecation block.